manala.ssh/templates/config/sshd/default.prod.j2

```diff
@@ -13,6 +13,9 @@
         ],
         'jessie': [
             '/etc/ssh/ssh_host_ed25519_key'
+        ],
+        'stretch': [
+            '/etc/ssh/ssh_host_ed25519_key'
```

Comment:
Same must apply for `default.dev` and `default.test`

Reply:
My bad.